MapboxDirections/MBRoute.swift

```diff
@@ -182,6 +182,13 @@ open class Route: NSObject, NSSecureCoding {
         }
         return nil
     }
+    
+    /**
+     A unique identifier for a directions request.
+     
+     Note, each `Route` in a request will have the same `routeIdentifier`.
```

Comment:
> Each route produced by a single call to `calculate(_:completionHandler:)` has the same route identifier.

Reply:
Can we say anything about how the developer uses this identifier? Or is it only for the navigation SDK’s use?